agent/lib/kontena/network_adapters/weave.rb

```diff
@@ -1,13 +1,11 @@
 require_relative '../logging'
-require_relative '../helpers/node_helper'
 require_relative '../helpers/iface_helper'
 require_relative '../helpers/weave_helper'
 
 module Kontena::NetworkAdapters
   class Weave
     include Celluloid
     include Celluloid::Notifications
-    include Kontena::Helpers::NodeHelper
```

Comment:
AFAICT the current class didn't use the `NodeHelper` methods (`master_url` or `node_id`).